detect/fixeddetector.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+from .detector import Detector
+from dataset import Table
```

Comment:
Please leave a blank line between python native packages, 3rd party packages, and project packages. Order each block alphabetically

Reply:
Done.